src/Implement/ResponseHandler.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <iostream>
+#include <sys/socket.h>
+
+#include "OperateData.h"
+#include "ResponseHandler.h"
+#include "ResponseEncoder.h"
+
+void ResponseHandler::Send(const std::string &for_client_fd, const std::unique_ptr<OperateData> &data) {
```

Comment:
When we pass unique_ptr, we use the raw unique_ptr because that denotes passing of ownership.

Using const or reference would break that passing of ownership, so that's why we don't add const or reference to unique_ptr when we are passing the ownership.